vhdl_lang/src/syntax/design_unit.rs

```diff
@@ -578,7 +599,8 @@ end;
             [simple_architecture(
                 WithDecl::new(code.s1("arch_name").ident()),
                 code.s1("myent").ident(),
-                None
+                None,
+                Position::new(1, 34).range_to(Position::new(2, 0)),
```

Comment:
I have really tried to avoid hard-coding positions in the test cases as it makes them very tedious to update. I have made a lot of utilities to find the position of substrings within the code that I always use.